tslearn/datasets.py

```diff
@@ -22,6 +22,32 @@
 
 __author__ = 'Romain Tavenard romain.tavenard[at]univ-rennes2.fr'
 
+def harmonize_dataset_filenames(folder, verbose=False):
+    """
+    Checks if tilename corresponds to the dataset name format and deletes .DS_Store files if exists
+
+        <UCR_cache>/
+            <datasetname>/
+                <datasetname>_TRAIN.txt
+                <datasetname>_TEST.txt
+                ...
+
+    :param folder: unzipped dataset folder
```

Comment:
Here, could you use `numpy` style docstring, as done in the rest of the tslearn package?

Reply:
Same comment for all your docstrings btw :)

Reply:
I fixed the doc strings in recent commits.
Have a look if that works out for you :-)
Unfortunately, I can't get the Cognitive Complexity below 5.
I assume that it is connected with the level of indentations. I added comments and more useful variable names, though. In my opinion `harmonize_dataset_filenames` is now quite readable.
However, I am open to more suggestions :-)